submit_a_new_code.md

```diff
@@ -8,10 +8,19 @@ comment: >-
 
 ### Requirements
 
-The main requirements for listing a code are that the code includes an
-[open source license](http://choosealicense.com){:target="_blank"}, is
-available via a public repository and includes annotated phase field
-examples.
+The two main requirements for including a code on the front page are
```

Comment:
The grammar before a list applies to each element; "... on the front page are and that" does not flow. Consider instead,
```
The two main requirements for including a code on the front page are that
1. at least one... and
2. the submitted benchmark...
```